### PR TITLE
Grasshopper components!

### PR DESCRIPTION
**Pull request by gonzalocasas**
This PR adds all the scaffolding needed to build Grasshopper components for COMPAS directly from source code (instead of binary blobs). It also adds one first component, but more as a proof of concept than anything else, the component outputs compas version and active environment.

This is how it looks:
image

Currently the components are built using a `pyinvoke` task and I added them to the ironpython build in Github Actions but (as agreed with Tom) we should change the release step of `pyinvoke` to release based on version tags and build the gh components in that step (see here for reference: https://github.com/compas-dev/compas_fab/pull/294).

### What type of change is this?

- [ ] Bug fix in a **backwards-compatible** manner.
- [x] New feature in a **backwards-compatible** manner.
- [ ] Breaking change: bug fix or new feature that involve incompatible API changes.
- [ ] Other (e.g. doc update, configuration, etc)

### Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I added a line to the `CHANGELOG.md` file in the `Unreleased` section under the most fitting heading (e.g. `Added`, `Changed`, `Removed`).
- [ ] I ran all tests on my computer and it's all green (i.e. `invoke test`).
- [x] I ran lint on my computer and there are no errors (i.e. `invoke lint`).
- [ ] I added new functions/classes and made them available on a second-level import, e.g. `compas.datastructures.Mesh`.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: src/compas_ghpython/__init__.py [hunk 1]
```diff
@@ -14,14 +14,40 @@
 """
 import os
 import compas
+import compas_rhino
 
+# TODO: I believe this should be removed, as it pulls all utilities funcs
```
Comment (tomvanmele) [inline src/compas_ghpython/__init__.py:19]: agreed. perhaps we can do 1 big cleanup for the release of 2.0

File: src/compas_ghpython/components/Compas_Info/code.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+Displays information about the active COMPAS environment.
+"""
+import compas_bootstrapper
+from ghpythonlib.componentbase import executingcomponent as component
+
+import compas
+
+
+class CompasInfo(component):
+    def RunScript(self):
```
Comment (tomvanmele) [inline src/compas_ghpython/components/Compas_Info/code.py:11]: this is actually very similar to how the python command plugins work for rhino (used for example in rv2) perhaps we should formalize that mechanism as well, and especially add a base plugin for rhino from which it is easier to do compas related infrastructure stuff. like a default compas menu for rhino...

**Review by tomvanmele — APPROVED** [review]

File: src/compas_ghpython/components/Compas_Info/code.py (hunk 2 above)
Comment (gonzalocasas) [reply src/compas_ghpython/components/Compas_Info/code.py:11]: Added it as topic for the next dev meeting!

File: src/compas_ghpython/__init__.py (hunk 1 above)
Comment (gonzalocasas) [reply src/compas_ghpython/__init__.py:19]: 👍

**Pull request merged**